cinema/views.py

```diff
@@ -35,7 +41,7 @@
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The permission class `IsAdminOrIfAuthenticatedReadOnly` is correctly applied to the `GenreViewSet`, ensuring that only admins can create genres while authenticated users can read them.